dyninstAPI/src/inst-x86.C

```diff
@@ -549,13 +549,17 @@ void registerSpace::initialize()
 int cpuidCall() {
     DWORD result = 0;
 // Note: mov <target> <source>, so backwards from what gnu uses
+#ifdef _WIN64
+	assert(0); // FIXME: Figure out a solution without inline asm.
```

Comment:
Excellent. GCC 4.8+ also has the `__builtin_cpu_supports` intrinsic: https://gcc.gnu.org/gcc-4.8/changes.html

ICC looks to have different intrinsics (though its gnu compatiblity mode may work? Don't know) and I haven't checked CLang.